src/templates/base/base_chapter.html

```diff
@@ -308,19 +301,42 @@ <h1 class="title title-lg">
               <source type="image/jpeg" srcset="{{ chapter_hero_dir }}hero_sm.jpg 1x, {{ chapter_hero_dir }}hero_lg.jpg 2x" />
               <img src="{{ chapter_hero_dir }}hero_lg.jpg" class="content-banner" alt="" width="866" height="433" loading="eager" />
             </picture>
+            <div id="cta-container">
+              <a class="alt btn chapter-cta comment-cta" href="https://discuss.httparchive.org/t/{{ metadata.get('discuss') }}">
+                <svg width="22" height="22" role="img">
+                  <title>{{ self.discuss_this_chapter() }}</title>
+                  <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#comment"></use>
+                </svg>
+                <span id="num_comments" aria-hidden="true">{{ self.comments_fallback() }}</span> <span data-translation id="comment-singular">{{ self.comment() }}</span>
+                <span data-translation id="comment-plural" aria-hidden="true">{{ self.comments() }}</span>
+              </a>
+              <a class="alt btn chapter-cta suggest-edit-cta" href="https://github.com/HTTPArchive/almanac.httparchive.org/issues/new?assignees=&labels=bug%2C+writing&template=2020-content-issue.md&title=Issue+with+the+2020+%5BCHAPTER%5D+chapter">
+                <svg width="22" height="22" role="img">
+                  <title>GitHub: Suggest Edit</title>
+                  <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#github"></use>
+                </svg>
+                <span aria-hidden="true">Suggest Edit</span>
+              </a>
+              <a class="alt btn chapter-cta help-translate-cta" href="https://github.com/HTTPArchive/almanac.httparchive.org/issues/923">
+                <svg width="22" height="22" role="img">
+                  <title>GitHub: Help Translate</title>
+                  <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#globe"></use>
+                </svg>
+                <span aria-hidden="false">Help Translate</span>
+              </a>
+            </div>
             {{ render_byline() }}
             {{ self.main_content() }}
+            <div class="chapter_links">
+              {{ render_actions() }}
+            </div>
+            <section class="authors">
+                {{ render_authors() }}
+            </section>
         </article>
-        <div class="chapter_links">
-          {{ render_actions() }}
-        </div>
-        <section class="authors">
-            {{ render_authors() }}
-        </section>
-
```

Comment:
I've reverted this. change and moved both the `chapter_links` and the `authors` back out of the article again in my version to allow the CTAs to stop at the right place.

I also had to add a spacer div for where the floating CTAs slot into.

```html
            </div>
            {{ render_byline() }}
            {{ self.main_content() }}
            <div class="spacer"></div>
        </article>
        <div class="chapter_links">
          {{ render_actions() }}
        </div>
        <section class="authors">
            {{ render_authors() }}
        </section>
```

And then added this class to almanac.css:

```css
.spacer {
  min-height: 60px;
}
```